dropshot/src/server.rs

```diff
@@ -327,7 +328,7 @@ async fn http_request_handle_wrap<C: ServerContext>(
         "uri" => format!("{}", request.uri()),
     ));
     trace!(request_log, "incoming request");
-    dropshot_request_start!(|| {
+    probes::request_start!(|| {
```

Comment:
This should work fine, once the provider module has been renamed as I mentioned above.